lib/Amazon/S3/Thin.pm

```diff
@@ -278,42 +278,46 @@ You can also pass any useragent as you like
 
 =head1 DESCRIPTION
 
-Amazon::S3::Thin - A thin, ligthweight, low-level Amazon S3 client.
+Amazon::S3::Thin is a thin, lightweight, low-level Amazon S3 client.
+If offers the following features:
```

Comment:
O_O look at that! I made a typo myself :(

This should read "It" and not "If". I have fixed it and amended the commit, but Github hasn't refreshed it yet. When you get a chance to review this PR, please check if this is fixed after the merge. Thanks! :)